### PR TITLE
Add AVRIT token metadata

### PR DESCRIPTION
**Pull request by amiyatulu**
# Add AVRIT token metadata
NEP-141: https://explorer.mainnet.near.org/accounts/avrit.near
Aurora ERC-20: https://explorer.mainnet.aurora.dev/address/0xB4530AA877D25E51c18677753ACd80fFa54009b2

### CODE REVIEW

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for the PR.
Added comment and also please make sure the description matches the template (check README).

**Comment by amiyatulu** [comment]
Thank you. Updated the logo with small size (5kb)

File: tokens/avrit.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
...wwLDMuNDVsMTctMi40MWMwLTEuMjMuMDctMi4zOS4xNy0zLjQ4Wm0tMTcsMTAuNDRhMTYuMjEsMTYuMjEsMCwwLDAsLjUzLDIuNTJsMTYuNDUtMmExMy42NywxMy42NywwLDAsMCwuMDYtMi43M2gwdi0uMTlsLTE3LDIuNDJabTIuODcsNi43OWE3LjU2LDcuNTYsMCwwLDAsMi4wOSwxLjU1LDcuNCw3LjQsMCwwLDAsMy42Ni43Miw4LjUzLDguNTMsMCwwLDAsMy43MS0xLjE0LDkuMDcsOS4wNywwLDAsMCwyLjg2LTIuNjRsLTEyLjMyLDEuNTFaTTExMC42OCw1MGEyLjgsMi44LDAsMCwxLDMuMDgsMi41LDIuODEsMi44MSwwLDAsMS0yLjUxLDMuMDhsLTguNjEuOTFhMi43OCwyLjc4LDAsMCwxLTIuMDYtLjYyLDIuODEsMi44MSwwLDAsMSwxLjQ5LTVsOC42MS0uOVpNOTQuNDgsMTVhMi44NSwyLjg1LDAsMCwxLDEuNzktMS4xOUEyLjgsMi44LDAsMCwxLDk5LjU3LDE2YTIuNzgsMi43OCwwLDAsMS0uNDIsMi4xbC00LjgxLDcuMmEyLjgsMi44LDAsMCwxLTEuNzksMS4yLDIuODQsMi44NCwwLDAsMS0yLjEtLjQzLDIuNzgsMi43OCwwLDAsMS0xLjItMS43OSwyLjc1LDIuNzUsMCwwLDEsLjQzLTIuMWw0LjgtNy4yWk01Ny4zLDIuNzRhMi44NSwyLjg1LDAsMCwxLC44Ni0yLDIuODEsMi44MSwwLDAsMSw0Ljc0LDIuMWwtLjIxLDguNjVhMi44MSwyLjgxLDAsMSwxLTUuNjEtLjEybC4yMi04LjY2Wk0xNCwxNi42NGwwLDBhMi44MSwyLjgxLDAsMCwxLS4wOS0zLjkybDAsMGEyLjc4LDIuNzgsMCwwLDEsMS45LS44NSwyLjg0LDIuODQsMCwwLDEsMiwuNzVjMi4wOSwyLDQuMjMsMy45NSw2LjMzLDUuOTFhMi44MywyLjgzLDAsMCwxLC45LDIsMi44NywyLjg3LDAsMCwxLS43NSwybDAsMGEyLjgxLDIuODEsMCwwLDEtMy45Mi4wOWMtMi0xLjkzLTQuMjUtNC4wOS02LjM1LTUuOTJaIi8+PC9zdmc+",
```
Comment (sept-en) [inline tokens/avrit.json:7]:
```suggestion
    "icon": "data:image/svg+xml,%3Csvg id='Layer_1' data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 113.77 122.88'%3E%3Ctitle%3Eidea%3C/title%3E%3Cpath d='M2.89,59.92a2.79,2.79,0,0,1-2-.75,2.84,2.84,0,0,1-.88-2,2.8,2.8,0,0,1,2.72-2.89l8.65-.3a2.83,2.83,0,0,1,2,.76,2.81,2.81,0,0,1-1.83,4.85l-8.66.29ZM68.1,40.42a6.12,6.12,0,0,1,2.14.35,8.27,8.27,0,0,1,3.61,2.5,8,8,0,0,1,1.89,4.31,7.84,7.84,0,0,1-.12,2.4,9.29,9.29,0,0,1,1.83,2.21,7.3,7.3,0,0,1,1,3.92,7,7,0,0,1-1.43,4,9.66,9.66,0,0,1-2.4,2.25,8.87,8.87,0,0,1-1.53,5.5A7.15,7.15,0,0,1,68.63,71a7,7,0,0,1-8.2,4.9,5.7,5.7,0,0,1-3.58-3A6.13,6.13,0,0,1,52.76,76a6.37,6.37,0,0,1-5.16-1.29A6.24,6.24,0,0,1,45.27,71a6.42,6.42,0,0,1-1.81-.34,8.33,8.33,0,0,1-3.54-2.43A8.13,8.13,0,0,1,38,64.1a7.56,7.56,0,0,1,.09-2.63,9.15,9.15,0,0,1-1.67-2,7.54,7.54,0,0,1-1.14-4,7.16,7.16,0,0,1,1.45-4.19A9.87,9.87,0,0,1,39,49.09c0-.27,0-.55,0-.82A8.77,8.77,0,0,1,41,43a6.9,6.9,0,0,1,4.28-2.55h0a5.07,5.07,0,0,1,.22-.76,6.6,6.6,0,0,1,2.87-3.34,6.22,6.22,0,0,1,4.39-.77,6,6,0,0,1,3.93,2.86,6.06,6.06,0,0,1,3.94-2.86,6.17,6.17,0,0,1,4,.58,6.79,6.79,0,0,1,2.88,2.76,6,6,0,0,1,.56,1.48Zm-9.83.89V68.82a1.38,1.38,0,0,1,.25.58c.39,2.4,1.43,3.56,2.61,3.88a4,4,0,0,0,3.31-.76,3.65,3.65,0,0,0,1.68-2.8c0-1.22-.88-2.61-3.28-3.83a1.33,1.33,0,1,1,1.21-2.36c2.66,1.35,4,3,4.52,4.71a4.72,4.72,0,0,0,2.37-1.87A6.29,6.29,0,0,0,72,62a1.33,1.33,0,0,1,.63-1.43,7.61,7.61,0,0,0,2.33-2,4.32,4.32,0,0,0,.91-2.5,4.73,4.73,0,0,0-.68-2.51,6.74,6.74,0,0,0-1.79-2A1.35,1.35,0,0,1,72.84,50a4.92,4.92,0,0,0,.25-2.18A5.44,5.44,0,0,0,71.82,45a5.63,5.63,0,0,0-2.43-1.7,3.51,3.51,0,0,0-1.37-.2,6.25,6.25,0,0,1-1.29,2.35,1.32,1.32,0,1,1-2-1.7,3,3,0,0,0,.49-3.54,4,4,0,0,0-1.73-1.65,3.53,3.53,0,0,0-2.29-.35c-1.13.23-2.22,1.17-2.9,3.11Zm6.78,9a1.33,1.33,0,0,1,1.56-2.15c.28.2.53.41.77.62a8.78,8.78,0,0,1,2.91,5.88,9.34,9.34,0,0,1-1.78,6.26c-.19.26-.41.52-.66.8a1.33,1.33,0,0,1-2-1.78,7.37,7.37,0,0,0,.47-.58,6.76,6.76,0,0,0,1.29-4.49,6.14,6.14,0,0,0-2-4.11,5.7,5.7,0,0,0-.55-.45ZM55.6,42.7h0l0,0h0l-.06-.1h0l0,0h0v0h0v0h0v0h0v0h0v0h0c-.64-2.59-1.89-3.78-3.18-4a3.63,3.63,0,0,0-2.52.47A3.92,3.92,0,0,0,48,40.62a3.28,3.28,0,0,0,1.18,3.62,1.33,1.33,0,0,1-1.81,1.95,6.75,6.75,0,0,1-2-3,4.36,4.36,0,0,0-2.33,1.56,6.11,6.11,0,0,0-1.36,3.66,7.25,7.25,0,0,0,.07,1.21,1.34,1.34,0,0,1-.62,1.31,8,8,0,0,0-2.27,2,4.52,4.52,0,0,0-.93,2.64,4.81,4.81,0,0,0,.75,2.56,6.43,6.43,0,0,0,1.67,1.79,1.34,1.34,0,0,1,.5,1.53,5.08,5.08,0,0,0-.23,2.32,5.52,5.52,0,0,0,1.3,2.78,5.74,5.74,0,0,0,2.38,1.65,3.93,3.93,0,0,0,1.09.2c.52-1.72,2-3.45,4.68-4.84a1.33,1.33,0,1,1,1.21,2.36c-2.52,1.29-3.47,2.77-3.47,4.07a3.42,3.42,0,0,0,1.41,2.61,3.77,3.77,0,0,0,3,.79c1.29-.27,2.54-1.46,3.18-4.06a1.35,1.35,0,0,1,.2-.44V42.7Zm-9.88,6.37A1.33,1.33,0,1,1,47.58,51a6.17,6.17,0,0,0-1.85,4.17,6.71,6.71,0,0,0,1.52,4.52,1.33,1.33,0,0,1-2.08,1.67A9.46,9.46,0,0,1,43.07,55a8.83,8.83,0,0,1,2.65-6Zm-1.14,53.11a2.26,2.26,0,0,1-.48-4.48c-1.29-7.33-4.82-11.36-8.55-15.63-2.9-3.32-5.91-6.77-8.29-11.85a35.77,35.77,0,0,1-3.58-15.54,36.57,36.57,0,0,1,4.38-16.62l.06-.1h0A31.55,31.55,0,0,1,43.5,24.51a34,34,0,0,1,16-2.29,36.7,36.7,0,0,1,15.28,4.91A31.55,31.55,0,0,1,88.13,42.25,33.57,33.57,0,0,1,90.65,54a34.15,34.15,0,0,1-2.59,13.8c-2.65,6.55-6.15,10.62-9.42,14.43-3,3.5-5.82,6.77-7.38,11.77a2.26,2.26,0,0,1-.21,4.43l-.76.11c-.14,1.09-.23,2.25-.27,3.5l.25,0a2.26,2.26,0,1,1,.62,4.48l-.77.11,0,.45h0a17.91,17.91,0,0,1,0,2.53l.13,0a2.27,2.27,0,0,1,.55,4.5l-1.68.2a13.51,13.51,0,0,1-5.83,6.91,12.73,12.73,0,0,1-5.58,1.7A11.59,11.59,0,0,1,52,121.73a12.11,12.11,0,0,1-5.26-5.11,2.25,2.25,0,0,1-1.22-1.74,2.2,2.2,0,0,1,.07-.89,19.64,19.64,0,0,1-.87-3.8h-.07a2.26,2.26,0,0,1-.62-4.48l.54-.07c0-1.23,0-2.38,0-3.47Zm3.73-5.08L66.7,94.49c1.58-6.67,5-10.67,8.74-15,3.06-3.56,6.33-7.36,8.71-13.26a29.94,29.94,0,0,0,2.3-12.12,29.2,29.2,0,0,0-2.23-10.28A27.26,27.26,0,0,0,72.64,30.76a32.33,32.33,0,0,0-13.51-4.35,29.89,29.89,0,0,0-14.05,2A27.43,27.43,0,0,0,31.76,40.07a32.35,32.35,0,0,0-3.87,14.64,31.57,31.57,0,0,0,3.18,13.73A44.91,44.91,0,0,0,38.72,79.3c4.21,4.82,8.19,9.38,9.59,17.8ZM66,99.16,48.78,101.6c0,1.09.07,2.24,0,3.45l17-2.41c0-1.23.07-2.39.17-3.48Zm-17,10.44a16.21,16.21,0,0,0,.53,2.52l16.45-2a13.67,13.67,0,0,0,.06-2.73h0v-.19l-17,2.42Zm2.87,6.79a7.56,7.56,0,0,0,2.09,1.55,7.4,7.4,0,0,0,3.66.72,8.53,8.53,0,0,0,3.71-1.14,9.07,9.07,0,0,0,2.86-2.64l-12.32,1.51ZM110.68,50a2.8,2.8,0,0,1,3.08,2.5,2.81,2.81,0,0,1-2.51,3.08l-8.61.91a2.78,2.78,0,0,1-2.06-.62,2.81,2.81,0,0,1,1.49-5l8.61-.9ZM94.48,15a2.85,2.85,0,0,1,1.79-1.19A2.8,2.8,0,0,1,99.57,16a2.78,2.78,0,0,1-.42,2.1l-4.81,7.2a2.8,2.8,0,0,1-1.79,1.2,2.84,2.84,0,0,1-2.1-.43,2.78,2.78,0,0,1-1.2-1.79,2.75,2.75,0,0,1,.43-2.1l4.8-7.2ZM57.3,2.74a2.85,2.85,0,0,1,.86-2,2.81,2.81,0,0,1,4.74,2.1l-.21,8.65a2.81,2.81,0,1,1-5.61-.12l.22-8.66ZM14,16.64l0,0a2.81,2.81,0,0,1-.09-3.92l0,0a2.78,2.78,0,0,1,1.9-.85,2.84,2.84,0,0,1,2,.75c2.09,2,4.23,3.95,6.33,5.91a2.83,2.83,0,0,1,.9,2,2.87,2.87,0,0,1-.75,2l0,0a2.81,2.81,0,0,1-3.92.09c-2-1.93-4.25-4.09-6.35-5.92Z'/%3E%3C/svg%3E",
```

**Comment by sept-en** [comment]
Thanks. Still, the description should be fixed before merging.

**Comment by amiyatulu** [comment]
@sept-en Added the description.

File: tokens/avrit.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "",
```
Comment (sept-en) [inline tokens/avrit.json:2]:
```suggestion
    "ethereum_address": "",
    "near_address": "avrit.near",
```

**Review by sept-en — APPROVED** [review]

**Comment by amiyatulu** [comment]
Hi, aurora is still not showing metadata for AVRIT
https://rainbowbridge.app/deploy
aurora_metadata

**Pull request merged**